### ISSUE TITLE
Create packaging Github Action

### ISSUE THREAD

**Issue by seanbsu** [cs481-ekh/s24-meshers#26]
References #20 

- [ ] use the github action template for packaging to pyPI
- [ ] set up configurations after researching necessary configurations to package the project for pip install
- [ ] test the action by creating a release in github and trying to pip install

**Comment by paulvxx** [comment]
- Attempted to fix some changes with previous yml script (Git Action)
- Added a build section to the Git Action
- Next planning on linking the build with pyPI so that I can pip install the package

[t:90]

**Comment by paulvxx** [comment]
- The process for creating a new python package is a bit more tedious then we expected. First we will (or at least someone will) need a pyPI account (test PyPI, an experimental version is okay too). 
- I managed to walk through a tutorial involving setting up a public pypackage successfully.
- I did run into a little bit of trouble with pip that I needed to fix, costing time. 

* For the future, we will need to change the directory of our project so that it matches up appropriately with the way the package builder expects it to.
* I plan to use Test pyPI instead of the actual pyPI. 

[t:120]
